public/js/components/modal.jsx

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var React = require('react');
+var gettext = require('utils').gettext;
+var cx = require('classnames');
+
+
+module.exports = React.createClass({
+
+  displayName: 'Modal',
+
+  propTypes: {
+    children: React.PropTypes.array.required,
```

Comment:
This should be `isRequired`. Hmm, too bad these create warnings not errors. If there is a flag in react to throw errors we should try it.